app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<header>
+  <%= link_to pages_splash_path, data: { turbo: false }  do %>
+    <%= inline_svg_tag "chevron_left.svg", class: "chevron" %>
+  <% end %>
+  <h1>sing up</h1>
```

Comment:
- [ ] @Zilola-Nazarova Kindly check the spelling of your Sign up, it's currently spelled wrongly.